content/docs/introducing-jsx.md

```diff
@@ -70,13 +71,13 @@ ReactDOM.render(
 
 [](codepen://introducing-jsx)
 
-We split JSX over multiple lines for readability. While it isn't required, when doing this, we also recommend wrapping it in parentheses to avoid the pitfalls of [automatic semicolon insertion](http://stackoverflow.com/q/2846283).
+Separamos o JSX em múltiplas linhas para melhorar a legibilidade. Mesmo que não seja requerido, quando fizer isso, também recomendamos cercá-lo com parênteses para evitar as armadilhas da [inserção automática de ponto-e-vírgula](http://stackoverflow.com/q/2846283).
```

Comment:
```suggestion
Separamos o JSX em múltiplas linhas para melhorar a legibilidade. Mesmo que não seja obrigatório, quando fizer isso, também recomendamos colocar dentro de parênteses para evitar as armadilhas da [inserção automática de ponto-e-vírgula](http://stackoverflow.com/q/2846283).
```

Reply:
Just opinion about _"cercá-lo"_, please tell me if you think it's best this way 😄
I believe with time we will get a better sense about what to use to translate those difficult terms.

Reply:
Yeah, that was a tough one. I tried "_envolver_" and "_circular_", but none of those seemed right. "_colocar dentro de_" makes sense, though.